build.sbt

```diff
@@ -56,6 +56,13 @@ lazy val root = (project in file(".")).
       scalaCheck % Test
     ),
 
+    // Soot dependency using Paderborn Nexus
+    resolvers ++= sootResolvers,
+    libraryDependencies += soot,
+    
+    // Alternative Soot dependency using direct nightly build jar
+    // libraryDependencies += soot from "https://soot-build.cs.uni-paderborn.de/nightly/soot/soot-trunk.jar",
+
```

Comment:
I have also tried this alternative that directly links to the nightly build jar.